src/commands/protocols/publish/receiver/submit-proofs-command.js

```diff
@@ -99,13 +135,7 @@ class SubmitProofsCommand extends EpochCommand {
                     hashFunctionId,
                     agreementData,
                     operationId,
-                );
-
-                that.operationIdService.emitChangeEvent(
-                    OPERATION_ID_STATUS.COMMIT_PROOF.SUBMIT_PROOFS_END,
-                    operationId,
-                    agreementId,
-                    epoch,
+                    assertionId,
```

Comment:
Missing `stateIndex`, not needed `assertionId`

Reply:
submit-proofs-command will be scheduled from calculate proofs command where we calculate stateIndex so we are not missing it. 
AssertionID should be removed we don't need it